tooling/debugger/ignored-tests.txt

```diff
@@ -5,5 +5,6 @@ macros
 reference_counts
 references
 regression_4709
+regression_7323
```

Comment:
Would be good for us to track better what functionality the debugger cannot handle. Just noting this as I the additional ignored test made me think of it.